music21/romanText/tsvConverter.py

```diff
@@ -227,15 +228,9 @@ def _changeRepresentation(self):
         >>> tabCd.representationType
         'DCML'
 
-        >>> tabCd.numeral
-        '#vii'
-
         >>> tabCd._changeRepresentation()
         >>> tabCd.representationType
         'm21'
-
-        >>> tabCd.numeral
-        'vii'
         '''
 
         if self.representationType == 'm21':
```

Comment:
I noticed the style guide says that `enum` is to be preferred to string identifiers. Don't know how important this is to the main developers...

Reply:
That could be changed, but that code is not introduced by this PR.

Reply:
yeah, I might do a big switch to our StrEnum for a lot of parts of the system before V8, so that can be done then.

Reply:
This is an AWESOME Example!  Now I get it.